Sites/GD2/Project 1/Styles.css

```
@@ -0,0 +1,105 @@
+img {
+    max-width: 100%
```

Remember to end all your declarations with a semicolon:
```suggestion
    max-width: 100%;
```

I'm not going to call this out throughout the stylesheet, but do your own review to update your declarations please 🙏🏻